modules/CmpdReg/src/client/src/MetaLot.js

```diff
@@ -431,7 +432,8 @@ $(function() {
 	    },
 
 	    newLot: function () {
```

Comment:
I'm a little confused on when this newLot function is used. I understand where the ones in `NewLotSuccess.js` and `Search.js` are used?

Reply:
I think this is tied to the "New Lot" button at the bottom of the Lot detail view of an existing lot:
![Screen Shot 2023-01-26 at 4 05 19 PM](https://user-images.githubusercontent.com/18313455/214977868-d2b96903-72e9-4229-938e-2134171915db.png)